@xen-orchestra/backups/_runners/_vmRunners/FullXapi.js

```diff
@@ -29,8 +29,10 @@ exports.FullXapi = class FullXapiVmBackupRunner extends AbstractXapi {
 
   async _copy() {
     const { compression } = this.job
+    const vm = this._vm
+    const vmSnapshot = this._exportedVm
```

Comment:
Semantically, this is incorrect as the exported VM might not be a snapshot (e.g. if the VM is halted).